usecases/MinimumWorkingExample/dodo.py

```diff
@@ -374,3 +386,101 @@ def task_upload_to_openbis():
+        #length = float(data['length'])
+        length = 100 # as suggested here :https://github.com/BAMresearch/LebeDigital/pull/152#discussion_r1187107430
+
+        def calibrate_it(path_exp_data=processed_data_emodulus_directory,path_calibrated_data=calibrated_data_directory,
+                         exp_name=f):
+            output = read_exp_data_E_mod(path=path_exp_data, exp_name=exp_name + '.csv',
+                                         length=length, diameter=diameter)
+            E_samples = estimate_youngs_modulus(experimental_data=output,
+                                                calibration_metadata={"E_loc": E_loc, "E_scale": E_scale},
+                                                calibrated_data_path=path_calibrated_data, mode=config['mode'])
+            # store samples as csv
+            np.savetxt(os.path.join(calibrated_data_expwise, f + '_calibrated_samples.csv'), E_samples, delimiter=',')
+        # current outputs
+        ## -- output from probeye. dont really need it but for the time being
+        owl_file = Path(calibrated_data_expwise, 'calibrationWorkflow' + f)
+        displ_list = Path(calibrated_data_expwise, 'displacement_list_' + f + '.dat')
+        force_list = Path(calibrated_data_expwise, 'force_list_' + f + '.dat')
+        #another_file = Path(calibrated_data_directory, 'joint_samples_compression_test_calibration.dat')
+        ## -- created by the calibration script
+        calibrated_data_file = Path(calibrated_data_expwise,f + '_calibrated_samples.csv')
+
+        yield {
+            'name': f'calibrate {f}',
+            'actions': [(calibrate_it,[processed_data_emodulus_directory,calibrated_data_expwise,f])],
+            'file_dep': [exp_metadata_path,Path(processed_data_emodulus_directory,f+'.csv')], # the file dependancies, the metadata and exp files.
+            'targets': [owl_file,displ_list,force_list,calibrated_data_file], # the files which are output for each calibration
+            'clean': [clean_targets]
+        }
+@create_after(executed='perform_calibration')
+def task_perform_prediction():
+    # create folder, if it is not there
+    Path(predicted_data_directory).mkdir(parents=True, exist_ok=True)
+
+    # setting for fast test, defining the list
+    if config['mode'] == 'cheap':
+        list_exp_name = [single_example_name]
+    else:  # go through all files
+        list_exp_name = os.listdir(
+            processed_data_emodulus_directory)
+        list_exp_name = [os.path.splitext(f)[0] for f in list_exp_name]  # split extension
+
+    for f in list_exp_name:
+        # choose calibrated data to be used for prediction
+        #calibrated_data = single_example_name
+        calibrated_data =f
+        calibrated_data_path = os.path.join(calibrated_data_directory,calibrated_data + '/' + calibrated_data+'_calibrated_samples.csv')
+        df = pd.read_csv(calibrated_data_path, header=None)
+        samples = df.iloc[:,0].tolist()
```

Comment:
doesn't this include the complete list of samples (including the initial ones for the burn in phase to calibrate the emcee sampling density)?

Reply:
No burn in ones are automatically dropped in emcee. It just contains the samples we need.